src/main/webapp/index.html

```diff
@@ -23,286 +23,260 @@
-                                   data-toggle="tab"
-                                   href="#tabs-text-3" role="tab" aria-controls="tabs-text-3" aria-selected="false">
-                                    <span class="display-4 py-2 px-3 text-roboto text-uppercase">volunteers</span>
-                                </a>
-                            </li>
-                        </ul>
-                    </div>
-                    <div class="card shadow mt-3">
-                        <div class="card-body">
-                            <div class="tab-content" id="myTabContent">
-                                <div class="tab-pane fade show active px-5 py-3" id="tabs-text-1" role="tabpanel"
-                                     aria-labelledby="tabs-text-1-tab">
-                                    <ul class="description text-justify text-dark font-weight-400">
-                                        <li>
-                                            Interact directly with industry experts and academics that are hard to reach
-                                            and
-                                            exclusive via OneLive to gain valuable perspective and insight on how to
-                                            customise and innovate a new career path for you.
-                                        </li>
-                                        <li>
-                                            1 to 1 premium mentoring by industry experts and academics across the world
-                                            Exposure to postgraduate, internship and job opportunities. 
-                                        </li>
-                                    </ul>
-                                </div>
-                                <div class="tab-pane fade px-5 py-3" id="tabs-text-2" role="tabpanel"
-                                     aria-labelledby="tabs-text-2-tab">
-                                    <ul class="description text-justify text-dark font-weight-400">
-                                        <li>Gain exposure to your work and projects.</li>
-                                        <li>Teaching/mentoring experience.</li>
-                                        <li>Ability to make an impact on the Sri Lankan education system.</li>
-                                        <li>Chance to connect with young talent in Sri Lanka.</li>
-                                    </ul>
-                                </div>
-                                <div class="tab-pane fade px-5 py-3" id="tabs-text-3" role="tabpanel"
-                                     aria-labelledby="tabs-text-3-tab">
-                                    <ul class="description text-justify text-dark font-weight-400">
-                                        <li>Develop In demand, skills needed for the job market when you graduate.</li>
+        </section>
+        <section class="new-statement text-center">
```

Comment:
@Suthan97 Can you please contain this statement in a container

Reply:
I did it. Please check 😊. I apologise for the delay.😔